delta/plugins/search/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/SearchSpec.scala

```diff
@@ -159,14 +170,28 @@ class SearchSpec
       esClient.bulk(bulkSeq, Refresh.WaitFor).accepted
     }
 
-    "search all indices" in {
-      val results = search.query(jobj"""{"size": 100}""", Query.Empty)(bob).accepted
-      extractSources(results).toSet shouldEqual projections.flatMap(createDocuments).toSet
+    "search all indices accordingly to Bob's acls" in {
+      val results = search.query(payload, queryParameters)(bob).accepted
+      extractSources(results).toSet shouldEqual allDocuments
+    }
+
+    "search only the project 1 index according to Alice's acls" in {
+      val results = search.query(payload, queryParameters)(alice).accepted
+      extractSources(results).toSet shouldEqual project1Documents
+    }
+
+    "search within an unknown suite" in {
+      search.query(Label.unsafe("xxx"), payload, queryParameters)(bob).rejectedWith[UnknownSuite]
+    }
+
+    "search within a suite as Bob" in {
+      val results = search.query(proj2Suite, payload, queryParameters)(bob).accepted
+      extractSources(results).toSet shouldEqual project2Documents
```

Comment:
I would propose:

`search all indexes on a user with full access`
`search a suite on a user with full access`
`search all indexes on a user with restricted access`
`search a suite on a user with restricted access`

You would need either 3 projects to achieve this I think

As a reader I don't know who "Bob" or "alice" are